fluss-common/src/main/java/org/apache/fluss/config/ConfigOptions.java

```diff
@@ -769,13 +786,19 @@ public class ConfigOptions {
                             "Size of the thread pool used in scheduling tasks to copy segments, "
                                     + "fetch remote log indexes and clean up remote log segments.");
 
+    /**
+     * @deprecated This option is deprecated. Please use {@link ConfigOptions#SERVER_IO_POOL_SIZE}
+     *     instead.
+     */
+    @Deprecated
     public static final ConfigOption<Integer> REMOTE_LOG_DATA_TRANSFER_THREAD_NUM =
```

Comment:
There is still a usage in the test code. We should update the test code to use the new config.